api/secrets/models/configuration_status.go

```diff
@@ -0,0 +1,39 @@
+package models
+
+import (
+	"fmt"
+)
+
+// ConfigurationStatus Enumeration of the statuses of configuration
+type ConfigurationStatus int
+
+const (
+	// Pending In configuration but not in cluster
+	Pending ConfigurationStatus = iota
+
+	// Consistent In configuration and in cluster
+	Consistent
+
+	// Orphan In cluster and not in configuration
+	Orphan
+
+	// External Secret value is in external resource, status unknown
+	External
+
+	numStatuses
+)
+
+func (p ConfigurationStatus) String() string {
+	return [...]string{"Pending", "Consistent", "Orphan", "External"}[p]
+}
+
+// GetStatusFromName Gets status from name
+func GetStatusFromName(name string) (ConfigurationStatus, error) {
```

Comment:
GetStatusFromName function is not used and can probably be removed

Reply:
I cannot find usage of any `GetStatusFromName` methods. I will remove them

Reply:
Also removed not used `GetStatusFromJobStatus` and its tests